### PR TITLE
voxelvolume

### PR DESCRIPTION
**Pull request by theolundqvist**
- Add voxelvolume class
- fix uniforms

### CODE REVIEW

File: src/EDAN35/util/GameObject.cpp
Comment (theolundqvist) [inline src/EDAN35/util/GameObject.cpp:1]: did some changes here first but we are not using this code anymore

File: src/EDAN35/util/Transform.cpp
Comment (theolundqvist) [inline src/EDAN35/util/Transform.cpp:1]: just code formatting

File: src/EDAN35/util/parametric_shapes.cpp
Comment (theolundqvist) [inline src/EDAN35/util/parametric_shapes.cpp:1]: createCube is new, rest is code formatting

File: src/core/node.hpp
Comment (theolundqvist) [inline src/core/node.hpp:1]: this is not used

**Comment by Hillowwold** [comment]
I only changed the _trace()_ in the _voxel.frag_. And now it is at FVTA_Win/voxel.frag(https://github.com/theolundqvist/edan35-high-performance-graphics/blob/FVTA_Win/voxel.frag) , which is just under the UNLICENCE. You can just down load it and replace your own voxel.frag.

**Pull request merged**